articles/extensions/authorization-extension/v2/index.md

```diff
@@ -1,358 +1,31 @@
 ---
 toc: true
-description: This page explains how to setup and manage the Authorization Extension v2.
+title: Authorization Extension
+description: Guidance on setting up and managing the Authorization Extension
 ---
-# Auth0 Authorization Extension v2
+# Authorization Extension
 
-The Auth0 Authorization Extension provides user authorization support in Auth0. Currently the extension supports authorizations for Users using Groups, Roles and Permissions.
+The Authorization Extension provides support for user authorization via Groups, Roles, and Permissions. You can define the expected behavior during the login process, and your configuration settings will be captured in a [rule](/rules) that's executed during runtime.
 
-## How to Install
+## Get Started
 
-First make sure you have a Client created that can support the Authorization extension. Supported client types for the Authorization extension are: **Native**, **Single Page Web Applications** and **Regular Web Applications**. Clients with no type assigned or non-interactive clients are not supported.
+Before you can use the extension, you'll need to install it, configure the rule controlling its behavior during login, and set up your user management.
 
-::: warning
-Installing this extension creates an `auth0-authz` client for your account. **Do not delete this client!** If you uninstall the extension at a later date, this client will be deleted automatically.
-:::
+* [Installing the Extension](/extensions/authorization-extension/v2/implementation/installation)
```

Comment:
This would look better if you used the imperative form https://github.com/auth0/docs/blob/master/STYLEGUIDE.md#gerunds